index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="first-navbar">
+            <a href="https://www.facebook.com"> <img src="./images/facebook_icon.png"> </a>
+            <a href="https://www.facebook.com"> <img src="./images/twitter_icon.png"> </a>
+        </div>
+        <nav>
+            <div class="logo"> <a href="index.html"></a>
+            </div>
```

Comment:
- [ ] This logo should not be visible on mobile view as seen in the design template. Please fix this

![image](https://github.com/alizaamir123/Capstone-project/assets/84156714/25c9a4bb-98c8-46be-b073-a886ae261dcc)